scripts/update-versions.sh

```diff
@@ -32,10 +35,20 @@ if [[ ! -d "$this_script_directory" ]]; then
   this_script_directory="$PWD"
 fi
 readonly pom_yml_file="$this_script_directory/../data/pom.yml" # locate jbake pom.yml value file
+readonly major_version=${new_release::1}
 
-sed -i -E "s/releaseDate: [0-9]+-[0-9]+-[0-9]+/releaseDate: $release_date/g" "$pom_yml_file"
-sed -i -E "s/7\.[0-9]+\.[0-9]+\.(Final|Beta[0-9]*|CR[0-9]*)/$new_release/g" "$pom_yml_file" # change only 7.x series for Drools
-sed -i -E "s/7\.[0-9]+\.[0-9]+\-SNAPSHOT/$new_snapshot/g" "$pom_yml_file" # change only 7.x series for Drools
+if [[ "$major_version" -ge "8" ]]; then
+  sed -i -E "s/releaseDate: [0-9]+-[0-9]+-[0-9]+/releaseDate: $release_date/g" "$pom_yml_file" # change only release date for 8.x series for Drools
+  sed -i -E "s/8\.[0-9]+\.[0-9]+\.(Final|Beta[0-9]*|CR[0-9]*)/$new_release/g" "$pom_yml_file" # change only 8.x series
+  sed -i -E "s/droolsReleaseNotes: .*/droolsReleaseNotes: https:\/\/issues.jboss.org\/secure\/ReleaseNote.jspa?projectId=12313021\&version=$new_releasenotes_number/g" "$pom_yml_file" # change only release notes for 8.x series and greater
```

Comment:
This procedure to automatically link the release notes is probably note ideal because it only check JIRA while we have now PR created using GH issues (i.e. https://github.com/kiegroup/drools/pull/4903 ) so I think we will need to move to a mechanism that produce the release notes from git commits (i.e. we can explore something like https://github.com/orhun/git-cliff ).

This is just a side note, I think it is outside the scope of this PR

@baldimir FYI

Reply:
We are working on release notes on kogito runtimes: https://github.com/kiegroup/kogito-runtimes/pull/2982
This should be expanded to Drools and other Kogito repositories if tested ok on 1.39 release
And we could add the support of GH issues